templates/definition/meter/deye-mi.yaml

```diff
@@ -0,0 +1,43 @@
+template: deye-mi
```

Comment:
```suggestion
template: deye-micro
```

Wir müssen nicht ganz so kryptisch sein?

Reply:
Die heissen bei Deye in der Doku so?

Reply:
Ah, das wusste ich nicht- hatte aufs micro geschaut.